README.md

```diff
@@ -63,3 +95,72 @@ yarn lint:fix
 ```shell
 yarn lint:check
 ```
+
+## 🛜 4. Deploy the app
+
+### 4.1 Deploy smart contract
+
+-   Edit the `packages/contracts/.env` after `yarn build`.
+-   Get your `ETH_PROVIDER_URL` from [infura](https://www.infura.io/), [alchemy](https://alchemy.com/), or other provider services.
+-   Get your `PRIVATE_KEY` and paste it in `.env` and start with `0x`.
+-   Run
+    ```sh
+    yarn contracts deploy --network custom
+    ```
+    from root directory
+
+### 4.2 Deploy to [Vercel](https://vercel.com/)
+
+Vercel is a Frontend Cloud. You can easily deploy the frontend and relay service with Vercel.
```

Comment:
Deploy ~~to [Vercel](https://vercel.com/)~~ the frontend

[Vercel](https://vercel.com/) is a Frontend Cloud..